modules/pam_mkhomedir/pam_mkhomedir.8.xml

```diff
@@ -91,8 +91,14 @@
         <listitem>
           <para>
             The file mode creation mask is set to
-            <replaceable>mask</replaceable>. The default value of mask is
-            0022.
+            <replaceable>mask</replaceable>. The default value of mask
```

Comment:
Please use something like:
```
           The default value of mask
            is 0022. If this option is not specified, then the permissions
            of the created user home directory is set to the value of
            <option>HOME_MODE</option> configuration item from
            <filename>/etc/login.defs</filename>. If there is no such
            configuration item then the value is computed from the
            value of <option>UMASK</option> in the same file. If
            there is no such configuration option either the default
            value of 0755 is used for the mode.
```

Reply:
Updated documentation for pam_mkhomedir and mkhomedir_helper